src/common/exception-filter/serviceException.ts

```diff
@@ -17,14 +17,18 @@ export class ServiceException extends Error {
     Error.captureStackTrace(this, this.constructor);
   }
 
-  static EntityNotFoundException = (message: string): ServiceException => {
-    return new ServiceException(message, 404);
-  };
-
   static BadRequestException = (message: string): ServiceException => {
     return new ServiceException(message, 400);
   };
 
+  static AuthException = (message: string): ServiceException => {
+    return new ServiceException(message, 401);
+  };
```

Comment:
AuthException might be a bit vague, because it could be 401 or 403, maybe you can just rename it to unauthorizedException ? 

```suggestion
  static UnauthorizedException = (message: string): ServiceException => {
    return new ServiceException(message, 401);
  };
```

Reply:
done!
I also created forbiddenException for 403 error.